### ISSUE TITLE
d2 chaos test: edge between boards

### ISSUE THREAD

**Issue by gavin-ts** [terrastruct/d2#752]
Ran into this random chaos test locally:
A side note but I think we have run chaos enough with random chars, I think we should change the default for IDs to just be random words or at least alphanumeric so we can more easily read what the problem here is. It seems to me like most errors chaos finds aren't related to strange character combinations (although this could be one itself haven't dug into it).

```
    --- FAIL: TestD2Chaos/#00 (0.00s)
        d2chaos_test.go:82: failed to create "\"cP\\n\x19*U<Sx\bH#\x05gE\x06u\".\"+hxZyS\x11hZT)wv\\$\x18\x11;N!i-sI4\v^)\x1f\" -> \"cP\\n\x19*U<Sx\bH#\x05gE\x06u\".\"+hxZyS\x11hZT)wv\\$\x18\x11;N!i-sI4\v^)\x1f\".\"\"": failed to recompile:
            "cP\n*U<SH#gEu": {
              "+hxZyShZT)wv\$;N!i-sI4
                                     ^)": {
                shape: cloud
                v: {shape: circle}
                "": {
                  shape: sequence_diagram
p]      5:-Qw[(S=Te-<Di hsJ0'8Tl3tMH/4nQ[Z]W": '"YG\$pCd0jS;42
h7]ZO]  nL*\$Jrw       PZ0&c9%  f61>\n|d:G%n)\n#7|@WfUzR^Zk&SG
<z\n73Cg"' {shape: rectangle}X\n8E;A4
                }
              }
              "N]": {shape: package}
              "+hxZyShZT)wv\$;N!i-sI4
                                     ^)" -> "+hxZyShZT)wv\$;N!i-sI4
                                                                   ^)".""
            }
            "cP\n*U<SH#gEu" -- "cP\n*U<SH#gEu": Kv
            "cP\n*U<SH#gEu" <-> "cP\n*U<SH#gEu"
            
            11:37: cannot create edges between boards
FAIL
```

**Comment by nhooyr** [comment]
I think I ran into something like this but then I pinned the test and it passed lol

@gavin-ts next time you see it can you save the goenc version so we can pin it?

**Comment by gavin-ts** [comment]
updated chaos to only use a-z up to 8 chars for ids and labels and found a simpler version:

```
 d2chaos_test.go:82: failed to create "iopouz <-> iopouz.\"\"": failed to recompile:
        iopouz: {
          shape: page
          atx: {shape: code}
          ""
        }
        nv: {shape: code}
        iopouz.atx -- iopouz
        iopouz <-> iopouz.atx: pcneaoj
        nv -- iopouz: owklh
        nv -> iopouz
        iopouz <-> iopouz.""
        
        11:11: cannot create edges between boards
```

**Comment by nhooyr** [comment]
Closed by #785
